assets/stylesheets/layouts/_header.scss

```diff
@@ -0,0 +1,193 @@
+.header {
+
+  // Make sure the header is taking up space in mobile.
+  @media only screen and (max-width: $screen-md) {
+    height: 3.375rem;
+  }
+
+  .container {
+    margin-top: .625rem;
+  }
+
+  // Get the mobile icons in the correct location with correct color.
+  a.hidden-md {
+    margin-top: 1.125rem;
+    svg {
+      color: $cr-black;
+    }
+  }
+
+  // Brand / Logo
+  .navbar-brand.logo {
+    margin-top: 0.25rem;
+    @media only screen and (max-width: $screen-md) {
+      position: absolute;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+    svg {
+      @include size(10.625rem, 1.5rem);
+      color: $cr-black;
+    }
+  }
+
+  // Main nav menu
+  .nav > li {
+    position: static;
+    // Sign in and search adjust to fit the height of the other items.
+    > a.btn {
+      padding-top: .8125rem;
+      padding-bottom: .8125rem;
+    }
+    > a.cta {
+      padding-top: .65625rem;
+      padding-bottom: .65625rem;
+    }
+    // Remove margins from all links in main nav.
+    > a {
+      margin: 0;
+    }
+    // Links are dark by default, with no bg
+    > a:not(.cta) {
+      color: $cr-gray-dark;
+    }
+    // Hover should alway show primary/light combo
+    > a:hover:not(.cta) {
+      color: $cr-white;
+      background-color: $nav-link-hover-bg;
+    }
+    // Open links have primary bg and white text
+    &.open > a:not(.cta) {
+      color: $cr-white;
+    }
+    // When the menu is closed, hovering behavior should take precedence
+    &:not(.open) > a:hover:focus:not(.cta) {
+      background-color: $nav-link-hover-bg;
+    }
+    // Unopened, focused links have grey bg
+    &:not(.open) > a:focus:not(.cta) {
+      background-color: #eeeeee;
+    }
+  }
+
+  // Dropdown menu container
+  .dropdown-menu {
+    margin-left: 0.5rem;
+    margin-right: 0.5rem;
+    padding: 1.25rem 0;
+    right: 0;
+    top: 2.5rem;
+    box-shadow: 0 6px 9pt $cr-gray-lighter;
+    border-radius: $border-radius-small;
+  }
+
+  // Dropdown menu lists
+  .dropdown-menu ul {
+    @extend .list-unstyled;
+    li a {
+      display: block;
+      padding: .1875rem 1.25rem;
+      color: $cr-gray-dark;
+      &:hover {
+        text-decoration: none;
+        background-color: #f5f5f5;
```

Comment:
You may want to use a Sass color function on one of our defined gray colors here. Also, I believe dropdown links can be defined using the Bootstrap variables. `$dropdown-link-color` and `$dropdown-link-hover-color` are available and may give you the effect you're looking for.